src/status_im/contexts/settings/wallet/network_settings/max_active_networks_sheet.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im.contexts.settings.wallet.network-settings.max-active-networks-sheet
+  (:require [quo.core :as quo]
+            [status-im.contexts.wallet.networks.core :as networks]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn hide-bottom-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn view
+  []
+  (let [customization-color (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/drawer-top
+      {:title           (i18n/label :t/max-active-networks-title
```

Comment:
How do I test network limits ? Since there are a max of 5 networks, and we have only 5 in testnet mode.